### ISSUE TITLE
"Domo for Office" Add-in tab does not appear after install 

### ISSUE THREAD

**Issue by chadaustin11** [OfficeDev/office-js#3272]
Users are not seeing the "Domo" tab in their office suite after installing the Domo Add-in. 

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Mac
* Host [Excel, Word, PowerPoint, etc.]: Excel, Word, PowerPoint, Outlook 
* Office version number: 16.71 
* Operating System: macOS 13.2.1
* Browser (if using Office on the web): Chrome 
 
## Expected behavior
After installing the add-in by clicking insert -> Get Add-ins -> Search "Domo for Office" -> Add, the Domo tab should appear as an option in the top menu 

## Current behavior
After installing the Domo for Office add-in, the Domo tab does not appear as an option in the top menu 

## Steps to reproduce

1. Click insert in the top menu 
2. Click "Get Add-Ins" 
3. Search "Domo"
4. Click "Add" for Domo for Office 
5. Check to see if Domo tab appears 

## Context
We notified our customers of the availability of our Domo for Office addins in MS Appsource on 3/29 and immediately started receiving feedback that the add-in is not working properley as they are not able to view the Domo tab after install. They can find the add-in in the "my Addins section after install to begin using it but expected the tab to appear. 

After investigating the issue the only solution we can find is to remove the version restrictions.  We are assuming by removing the version restrictions, the app will now be available for version it's not going to work on, so any assistance to help resolve this issue will be appreciated.

**Comment by chrisdomo** [comment]
Also of note is the Domo tab does show up in the web version of the applications and not the desktop versions.

**Comment by dgpham112** [comment]
@chadaustin11 @chrisdomo 
more info: 
the addin is use for Word/Excel/Powerpoint so the manifest have multiple Host element
 ` <Hosts> 
<Host Name="Document" />
	<Host Name="Workbook" />
	<Host Name="Presentation" />
  </Hosts>`
  
The Requirements tag was placed in <VersionOverrides> tag
`  <VersionOverrides xmlns="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="VersionOverridesV1_0">
    <Requirements>
      <Sets DefaultMinVersion="1.1" xmlns="http://schemas.microsoft.com/office/officeappbasictypes/1.0">
        <Set Name="ExcelApi" MinVersion="1.8"/>
        <Set Name="WordApi" MinVersion="1.2"/>
        <Set Name="PowerPointApi" MinVersion="1.5"/>
		<Set Name="SharedRuntime" MinVersion="1.1" />
      </Sets>
    </Requirements>
    ....`

For Word, if we keep only Set tag for Word
` <Set Name="WordApi" MinVersion="1.2"/>`
 
 The ribbon tab show up properly.

The same for Powerpoint and Excel.

So it looks like that when use combined manifest, currently it does not allow Requirements Set for multiple platforms (Word,Excel/Powerpoint)

**Comment by dgpham112** [comment]
Ribbon tab for Domo add-in not show up if manifest contains multiple Requirements Set tag
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/7466336/229675276-cd422831-9cbb-4934-a1ed-2c96c140f21c.png">

If we keep only WordApi Requirement Set tag, ribbon tab show up when install add-in for Word. 
It is the same for Excel/Powerpoint if we only keep the tag that applicable to that platform.
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/7466336/229675442-4004a12f-2a2a-4c2b-a205-417410e1b8ef.png">

**Comment by JinghuiMS** [comment]
> @chadaustin11 @chrisdomo more info: the addin is use for Word/Excel/Powerpoint so the manifest have multiple Host element ` <Hosts> <Host Name="Document" /> <Host Name="Workbook" /> <Host Name="Presentation" /> </Hosts>`
> 
> The Requirements tag was placed in tag ` <VersionOverrides xmlns="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="VersionOverridesV1_0"> <Requirements> <Sets DefaultMinVersion="1.1" xmlns="http://schemas.microsoft.com/office/officeappbasictypes/1.0"> <Set Name="ExcelApi" MinVersion="1.8"/> <Set Name="WordApi" MinVersion="1.2"/> <Set Name="PowerPointApi" MinVersion="1.5"/> <Set Name="SharedRuntime" MinVersion="1.1" /> </Sets> </Requirements> ....`
> 
> For Word, if we keep only Set tag for Word ` <Set Name="WordApi" MinVersion="1.2"/>`
> 
> The ribbon tab show up properly.
> 
> The same for Powerpoint and Excel.
> 
> So it looks like that when use combined manifest, currently it does not allow Requirements Set for multiple platforms (Word,Excel/Powerpoint)

@dgpham112 you're right. multiple requirement sets of multiple hosts defined in the manifest are not supported. Thanks very much for your explanation!

For this scenario, a better strategy would be to use the technique described in [Runtime checks for method and requirement set support](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/specify-office-hosts-and-api-requirements#runtime-checks-for-method-and-requirement-set-support).

**Comment by amagupta-collab** [comment]
@chadaustin11 I have tried to install domo addin in excel mac current version (16.104) and ribbon tab is appearing. Can you please confirm if the issue is fixed
